core/pom.xml

```diff
@@ -8,17 +8,9 @@
     <version>8.1.0.0-SNAPSHOT</version>
   </parent>
   <artifactId>cpf-core</artifactId>
+  <packaging>bundle</packaging>
   <version>8.1.0.0-SNAPSHOT</version>
-  <properties>
-    <dependency.jackson-core-asl.revision>1.8.2</dependency.jackson-core-asl.revision>
```

Comment:
Because the dependency management section is a mechanism for centralizing dependency information. It's very helpful to have the information in a single place instead of spread through the several child POMs.
https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management

Reply:
The question is not what the dependency management element of maven is for. The question was why did you centralized all dependencies on the root. It's probably easier to have a real time chat on pros / cons of that approach.